styles.css

```diff
@@ -527,6 +514,131 @@ ul {
     color: #5e6c84;
     border: 1px solid #c1c7d0;
   }
+
+  #popup {
+    display: none;
+  }
+
+  .popup {
+    display: flex;
+    border-radius: 16px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    width: 80%;
+    z-index: 100;
+    background-color: white;
+    position: fixed;
+    top: 50px;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    margin: 0 auto;
+  }
```

Comment:
- [ ]  Please kindly increase the margin-bottom of your pop-up window for mobile version as well and kindly ensure that all the elements are properly displayed (perhaps you can reduce the size of the image) and aligned according to the template. 👍

## Your Design 
<img width="290" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225602030-37134177-c96c-4b55-8f90-c099333f81a2.png">

## Figma Design
<img width="114" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225602306-1efa2afa-2cf5-464b-a1a5-411c642f71d3.png">